provider/pprof/README.md

```diff
@@ -0,0 +1,24 @@
+# [pprof](https://github.com/google/pprof) context provider for OpenCtx
+
+This is a context provider for [OpenCtx](https://openctx.org) that annotates functions with the CPU time and memory allocations attributed to them.
+
+## Usage
+
+Add the following to your settings in any OpenCtx client:
```

Comment:
Would this provider work in browser environments?

Reply:
No, I don't think so. The provider expects to find the pprof report somewhere on your local machine, which the others using the extension would probably not have on their machines.

I'm going to update the docs today 👌